src/ZenstruckFoundryBundle.php

```diff
@@ -284,6 +293,12 @@ public function loadExtension(array $config, ContainerConfigurator $configurator
         $configurator->import('../config/in_memory.php');
 
         $container->registerForAutoconfiguration(InMemoryRepository::class)->addTag('foundry.in_memory.repository');
+
+        if ($config['persistence']['flush_once'] === false) {
```

Comment:
So what's the plan if some users have issues with making `true`? Try and fix the use-case? If we get too many issues, allow false?

Reply:
> Try and fix the use-case

yes

> If we get too many issues, allow false?

why not, as you can see, the change to have "flush once" is really small. But I'd would really prefer to support only one way to do this :crossed_fingers: 

I think that 95% of the time this will be OK. Some of the "bugs" could not be fixed, mainly because they won't be bugs but subtle changes in the behavior. the example with "soft-delete" I told you about is a perfect example. But it was quite complex to understand what was happening :grimacing: